compiler/resolution/callDestructors.cpp

```diff
@@ -1205,6 +1205,57 @@ static void destroyFormalInTaskFn(ArgSymbol* formal, FnSymbol* taskFn) {
   downEndCount->insertBefore(autoDestroyCall);
 }
 
+static void convertRawPointerTypes() {
```

Comment:
How about call it convertClassTypesToCanonical() ?

Also I welcome factoring out the common pattern, so the loops in this function look like:

```c++
forv_Vec(VarSymbol, var, gVarSymbols)
  convertHelp(var->type); // whatever you call the helper
```